package/scripts/presto_client.py

```diff
@@ -80,7 +80,8 @@ def ensure_nodes_are_up(client):
         raise RuntimeError('Presto server failed to start within \
 {0} seconds.'.format(RETRY_TIMEOUT))
 
-
+# This class was copied more or less verbatim from
```

Comment:
just curious, why are you not checking in the ensure_* methods above in this file for the exact number of nodes or exact type of connectors which are active?